metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py

```diff
@@ -488,22 +527,18 @@ def loop_tables(
             dataset_snapshot.aspects.append(schema_metadata)
 
             # add table properties
-            default_properties = ["create_date", "table_type", "table_location"]
-            properties: Dict[str, str] = {}
-            for prop in default_properties + self.config.extra_properties:
-                if prop in columns[-1]:
-                    properties[prop] = str(columns[-1][prop]) or ""
+            properties: Dict[str, str] = properties_cache.get(dataset_name, {})
+            properties["table_type"] = str(columns[-1]["table_type"]) or ""
+            properties["table_location"] = str(columns[-1]["table_location"]) or ""
```

Comment:
I wonder if we should check the value of `columns[-1]["xxx"]` before stringification.
str(None) = 'None' (a truthy value)